CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2.1.2
+
+## Enhancements
+
+* Bump scala sdk to 2.3.0 to replace vault token with k8s service token
```

Comment:
"Cognite Scala SDK", type out "Kubernetes". Maybe highlight that this is a Cognite internal change that doesn't matter to normal users.

Reply:
comment is updated 😁